src/test/pybind/test_cephfs.py

```diff
@@ -610,10 +610,10 @@ def test_ftruncate(testdir):
 def test_fallocate(testdir):
     fd = cephfs.open(b'/file-fallocate', 'w', 0o755)
     assert_raises(TypeError, cephfs.fallocate, b'/file-fallocate', 0, 10)
-    cephfs.fallocate(fd, 0, 10)
+    assert_raises(cephfs.OperationNotSupported, cephfs.fallocate, fd, 0, 10)
```

Comment:
This should be `libcephfs.OperationNotSupported`

Reply:
fixed the exception name